react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
Not responsive text without the baseline correction. This is the bandaid that `.touchableText` was striving for but that also missed the mark